notebooks/21_kaggle_data_cleaning.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<p>could you use <code>dm_dataset[0].media.data</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/1534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Fix in [3c2d3c1](https://github.com/openvinotoolkit/datumaro/pull/1534/commits/3c2d3c1b0b0d0f7457dabdd90398dd8a00bb04e9)